stdlib/lang-parser/parserModule/tests/parser_unit_tests.bal

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/test;
+import ballerina/log;
+
+//invoke parser to convert to json
+function parseAst(string fileLocation) returns json {
+    PackageNode? pkNode = parseFile(fileLocation);
+    if (pkNode is PackageNode) {
+        json | error jsonOut = json.convert(pkNode);
+        if (jsonOut is json) {
```

Comment:
Shall we do a `test:assertFail()` if `.convert()` returns an error or `parseFile()` returns `()` without relying on the `()` return here to cause the test failure when asserting equality? That way we'll know what exactly caused the failure?